### PR TITLE
Enable circleci integration tests

### LINKED ISSUE #9062: CircleCI tests are failing because of FxA
Unfortunately, CircleCI tests are failing currently because of https://github.com/mozilla/pytest-fxa/issues/15

### PR DESCRIPTION
**Pull request by jrbenny35**
Temp fix to get the integration-tests running. Should fix #9062 as the branch I am using is pretty much what the official release will be.

### CODE REVIEW

**Comment by jrbenny35** [comment]
I should add that if one tests gets an error we have better clean up now, so it should only error once and the next run it should be okay. It shouldn't error over and over like they were doing.

**Comment by EnTeQuAk** [comment]
You need to upgrade pluggy and potentially a few other dependencies that I updated as part of #9076 to get the tests working again.

File: requirements/uitests.txt [hunk 1]
```diff
@@ -71,8 +71,9 @@ zope.interface==4.5.0 \
     --hash=sha256:e6b77ae84f2b8502d99a7855fa33334a1eb6159de45626905cb3e454c023f339 \
     --hash=sha256:a1daf9c5120f3cc6f2b5fef8e1d2a3fb7bbbb20ed4bfdc25bc8364bc62dcf54b \
     --hash=sha256:57c38470d9f57e37afb460c399eb254e7193ac7fb8042bd09bdc001981a9c74c
-pluggy==0.6.0 \
-    --hash=sha256:7f8ae7f5bdf75671a718d2daf0a64b7885f74510bcd98b1a0bb420eb9a9d0cff
+    pluggy==0.7.1 \
+        --hash=sha256:6e3836e39f4d36ae72840833db137f7b7d35105079aee6ec4a62d9f80d594dd1 \
+        --hash=sha256:95eb8364a4708392bae89035f45341871286a333f749c3141c20573d2b3876e1
```
Comment (EnTeQuAk) [inline requirements/uitests.txt]: that indentation looks very wrong

Comment (jrbenny35) [reply requirements/uitests.txt]: wow that is...weird. Nothing file code editors.

**Comment by EnTeQuAk** [comment]
Thanks for this, @jrbenny35 can you open a new issue though to follow-up on this and make sure the upstream patch lands asap and get's merged into addons-server?

**Pull request merged**